cmd/replace_test.go

```diff
@@ -14,6 +14,8 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
+//go:build unit
```

Comment:
TIL: If a file does not define a tag, then it'll always be picked up. This is probably why you added the `unit` tag here. So it doesn't get executed when all we want to run are the acceptance tests.